src/pages/[slug].tsx

```diff
@@ -19,50 +30,59 @@ export const getStaticProps: GetStaticProps = async (context) => {
   };
 };
 
+export const getStaticPaths: GetStaticPaths = async () => {
+  const paths = await getProjectsSlugs();
+  return {
+    paths,
+    fallback: false,
```

Comment:
we need to have revalidate enabled with `fallback` set to "blocking"
so that we update generated pages frequently
to learn more: https://nextjs.org/docs/api-reference/data-fetching/get-static-paths#fallback-blocking